examples/src/main/python/ml/simple_params_example.py

```diff
@@ -18,36 +18,30 @@
 from __future__ import print_function
 
 import pprint
-import sys
 
-from pyspark import SparkContext
 from pyspark.ml.classification import LogisticRegression
 from pyspark.mllib.linalg import DenseVector
-from pyspark.mllib.regression import LabeledPoint
-from pyspark.sql import SQLContext
+from pyspark.sql import SparkSession
 
 """
-A simple example demonstrating ways to specify parameters for Estimators and Transformers.
+An example demonstrating ways to specify parameters for Estimators and Transformers.
 Run with:
   bin/spark-submit examples/src/main/python/ml/simple_params_example.py
 """
 
 if __name__ == "__main__":
-    if len(sys.argv) > 1:
-        print("Usage: simple_params_example", file=sys.stderr)
-        exit(1)
-    sc = SparkContext(appName="PythonSimpleParamsExample")
-    sqlContext = SQLContext(sc)
+    spark = SparkSession \
```

Comment:
I wonder this rxample works. I remember this was not fixed in https://github.com/apache/spark/pull/12809 because it does not work.